### PR TITLE
Add Script to Generate Test Files

### PR DESCRIPTION
**Pull request by jxguan**

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func (c *Client) getFile(docID int) {
 
 // SearchWord searches for a word in all the documents and returns the names of
 // all the documents containing that word as a string slice.
-func (c *Client) SearchWord(word string) []string {
+func (c *Client) SearchWord(word string) ([]string, float64) {
```
Comment (strib) [inline client/client.go]: Please document in the comment what the second return value.

File: test/testfile.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+var dictFile = flag.String("dict_file", "dictionary.txt", "The dictionary file used to generate the random words")
+var outputPath = flag.String("output_path", "testFiles", "The directory where the test files should be stored")
+
+// This is a little tool for creating test files with random English words.  Use
+// `go run testFile.go --help` to check the configurable parameters.
+func main() {
+	if _, err := os.Stat(*outputPath); os.IsNotExist(err) {
+		if os.Mkdir(*outputPath, 0777) != nil {
+			fmt.Println("Failed to create the output directory", *outputPath)
+			return
```
Comment (strib) [inline test/testfile.go]: Whenever there's a fatal error, you should probably do an `os.Exit(-1)`.

**Comment by strib** [comment]
Looks good mod nits, thanks!

One question: where did you get `dictionary.txt`?  It might be good to attribute it in the README or in a comment in the file itself.

File: client/client.go (hunk 1 above)
Comment (jxguan) [inline client/client.go]: Fixed

File: test/testfile.go (hunk 2 above)
Comment (jxguan) [inline test/testfile.go]: Fixed

**Pull request merged**